### ISSUE TITLE
 EXCEPTION_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by RossBoylan** [JuliaLang/julia#33959]
I got the error in the subject along with instructions to report it.  This is far from a minimal reproducible example.  My first use of newly installed julia 1.3.0.  

In broad strokes:
1. Installed Julia 1.3.0 and, with [considerable difficulty](https://discourse.julialang.org/t/temporarily-unable-to-edit-path-for-julia-in-julia-client/31558), got atom to use the new Julia.
2. Started REPL in atom.
3. Switched to an environment I previously used in 1.2.0.
4. updated the packages
5. Attempted to execute some test code that previously ran in 1.2.0.  Executed in blocks
     a) `using` directives for various libraries
     b) read in some data
     c) define a variable as a list of symbols.  That's where it failed.

Here's the transcript, with some comments about what I did.
```
Press Enter to start Julia. 

# Step 2 of my broad strokes description

Hold on tight while we are installing some packages for you.
This should only take a few seconds...

Activating environment at `C:\Users\rdboylan\.julia\environments\v1.3\Project.toml`
  Updating registry at `C:\Users\rdboylan\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed Parsers ──────── v0.3.10
 Installed MacroTools ───── v0.5.2
 Installed JuliaFormatter ─ v0.1.19
 Installed HTTP ─────────── v0.8.8
 Installed Atom ─────────── v0.11.3
 Installed DataStructures ─ v0.17.6
  Updating `C:\Users\rdboylan\.julia\environments\v1.3\Project.toml`
  [c52e3926] + Atom v0.11.3
  Updating `C:\Users\rdboylan\.julia\environments\v1.3\Manifest.toml`
  [bf4720bc] + AssetRegistry v0.1.0
  [c52e3926] + Atom v0.11.3
  [b99e7846] + BinaryProvider v0.5.8
  [00ebfdb7] + CSTParser v0.6.2
  [7057c7e9] + Cassette v0.2.6
  [53a63b46] + CodeTools v0.6.4
  [da1fd8a2] + CodeTracking v0.5.8
  [3da002f7] + ColorTypes v0.8.0
  [5ae59095] + Colors v0.9.6
  [34da2185] + Compat v2.2.0
  [864edb3b] + DataStructures v0.17.6
  [b4f34e82] + Distances v0.8.2
  [33d173f1] + DocSeeker v0.3.1
  [53c48c17] + FixedPointNumbers v0.6.1
  [de31a74c] + FunctionalCollections v0.5.0
  [cd3eb016] + HTTP v0.8.8
  [9fb69e20] + Hiccup v0.2.2
  [83e8ac13] + IniFile v0.5.0
  [682c06a0] + JSON v0.21.0
  [98e50ef6] + JuliaFormatter v0.1.19
  [aa1ae85d] + JuliaInterpreter v0.7.4
  [e5e0dc1b] + Juno v0.7.2
  [7c4cb9fa] + LNR v0.2.0
  [50d2b5c4] + Lazy v0.14.0
  [1914dd2f] + MacroTools v0.5.2
  [739be429] + MbedTLS v0.7.0
  [e89f7d12] + Media v0.5.0
  [510215fc] + Observables v0.2.3
  [bac558e1] + OrderedCollections v1.1.0
  [69de0a69] + Parsers v0.3.10
  [fa939f87] + Pidfile v1.1.0
  [189a3867] + Reexport v0.2.0
  [ae029012] + Requires v0.5.2
  [88034a9c] + StringDistances v0.4.0
  [0796e94c] + Tokenize v0.5.6
  [37b6cedf] + Traceur v0.3.0
  [a2a6695c] + TreeViews v0.3.0
  [30578b45] + URIParser v0.4.0
  [0f1e0344] + WebIO v0.8.11
  [104b5d7c] + WebSockets v1.5.2
  [cc8bc4a8] + Widgets v0.6.2
  [2a0f44e3] + Base64 
  [ade2ca70] + Dates 
  [8bb1440f] + DelimitedFiles 
  [8ba89e20] + Distributed 
  [7b1f6079] + FileWatching 
  [b77e0a4c] + InteractiveUtils 
  [76f85450] + LibGit2 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [a63ad114] + Mmap 
  [44cfe95a] + Pkg 
  [de0858da] + Printf 
  [9abbd945] + Profile 
  [3fa0cd96] + REPL 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [1a1011a3] + SharedArrays 
  [6462fe0b] + Sockets 
  [2f01184e] + SparseArrays 
  [10745b16] + Statistics 
  [8dfed614] + Test 
  [cf7118a7] + UUIDs 
  [4ec0a83e] + Unicode 
 Resolving package versions...
  Updating `C:\Users\rdboylan\.julia\environments\v1.3\Project.toml`
  [e5e0dc1b] + Juno v0.7.2
  Updating `C:\Users\rdboylan\.julia\environments\v1.3\Manifest.toml`
 [no changes]

Starting Julia...
[ Info: Precompiling Atom [c52e3926-4ff0-5f6e-af25-54175e0327b1]
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.3.0 (2019-11-26)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

# Step 3
(v1.3) pkg> activate C:\Users\rdboylan\Documents\KBD2\MyFirst
Activating environment at `C:\Users\rdboylan\Documents\KBD2\MyFirst\Project.toml`

(MyFirst) pkg> status
Project MyFirst v0.1.0
    Status `C:\Users\rdboylan\Documents\KBD2\MyFirst\Project.toml`
  [324d7699] CategoricalArrays v0.7.1
  [a93c6f00] DataFrames v0.19.4
  [31c24e10] Distributions v0.19.2
  [f6369f11] ForwardDiff v0.10.6
  [38e38edf] GLM v1.1.2 [`dev\GLM`]
  [54eb57ff] InteractiveCodeSearch v0.3.1
  [d41bc354] NLSolversBase v7.5.0
  [86f7a689] NamedArrays v0.9.3
  [429524aa] Optim v0.19.4
  [c46f51b8] ProfileView v0.5.0
  [6f49c342] RCall v0.13.4
  [df47a6cb] RData v0.6.3
  [295af30f] Revise v2.2.2
  [1463e38c] StatFiles v0.8.0
  [2913bbd2] StatsBase v0.30.0
  [3eaba693] StatsModels v0.6.5 [`dev\StatsModels`]
  [bd369af6] Tables v0.2.11
  [37e2e46d] LinearAlgebra

# Step 4
(MyFirst) pkg> update
  Updating registry at `C:\Users\rdboylan\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed Calculus ────────── v0.5.1
 Installed DiffEqDiffTools ─── v1.4.0
 Installed Rmath ───────────── v0.6.0
 Installed LibExpat ────────── v0.6.0
 Installed Polynomials ─────── v0.6.0
 Installed EzXML ───────────── v0.9.5
 Installed CategoricalArrays ─ v0.7.3
 Installed Revise ──────────── v2.3.1
 Installed LoweredCodeUtils ── v0.4.2
 Installed NaNMath ─────────── v0.3.3
 Installed TableTraitsUtils ── v1.0.1
 Installed Expat_jll ───────── v2.2.7+0
 Installed FileIO ──────────── v1.1.0
 Installed TimeZones ───────── v0.10.3
  Updating `C:\Users\rdboylan\Documents\KBD2\MyFirst\Project.toml`
  [324d7699] ↑ CategoricalArrays v0.7.1 ⇒ v0.7.3
  [295af30f] ↑ Revise v2.2.2 ⇒ v2.3.1
  Updating `C:\Users\rdboylan\Documents\KBD2\MyFirst\Manifest.toml`
  [49dc2e85] ↑ Calculus v0.5.0 ⇒ v0.5.1
  [324d7699] ↑ CategoricalArrays v0.7.1 ⇒ v0.7.3
  [864edb3b] ↑ DataStructures v0.17.5 ⇒ v0.17.6
  [01453d9d] ↑ DiffEqDiffTools v1.3.0 ⇒ v1.4.0
  [2e619515] + Expat_jll v2.2.7+0
  [8f5d6c58] ↑ EzXML v0.9.4 ⇒ v0.9.5
  [5789e2e9] ↑ FileIO v1.0.7 ⇒ v1.1.0
  [522f3ed2] ↑ LibExpat v0.5.0 ⇒ v0.6.0
  [6f1432cf] ↑ LoweredCodeUtils v0.3.8 ⇒ v0.4.2
  [77ba4419] ↑ NaNMath v0.3.2 ⇒ v0.3.3
  [69de0a69] ↑ Parsers v0.3.8 ⇒ v0.3.10
  [f27b6e38] ↑ Polynomials v0.5.3 ⇒ v0.6.0
  [295af30f] ↑ Revise v2.2.2 ⇒ v2.3.1
  [79098fc4] ↑ Rmath v0.5.1 ⇒ v0.6.0
  [382cd787] ↑ TableTraitsUtils v1.0.0 ⇒ v1.0.1
  [f269a46b] ↑ TimeZones v0.10.2 ⇒ v0.10.3
  Building EzXML ────→ `C:\Users\rdboylan\.julia\packages\EzXML\QtGgF\deps\build.log`
  Building Rmath ────→ `C:\Users\rdboylan\.julia\packages\Rmath\BoBag\deps\build.log`
  Building TimeZones → `C:\Users\rdboylan\.julia\packages\TimeZones\pjvlM\deps\build.log`

# Step 5a
[ Info: Precompiling MyFirst [074633b0-7e9c-11e9-0fe2-650315f203db]
[ Info: Precompiling StatFiles [1463e38c-9381-5320-bcd4-4134955f093a]
[ Info: Precompiling ProfileView [c46f51b8-102a-5cf2-8d2c-8597cb0e0da7]
[ Info: Precompiling Gtk [4c0ca9eb-093a-5379-98c5-f87ac0bbbf44]
[ Info: Precompiling GtkReactive [27996c0f-39cd-5cc1-a27a-05f136f946b6]

#Steps 5b and then c

Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety).
Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x7ff8c170a229 --
```

Here's the script I was running
```julia

# Step 5a
using MyFirst
using DataFrames
using StatFiles
using StatsBase
using Statistics
using StatsModels
using Profile
using ProfileView

# step 5b
nh = DataFrame(load("C:\\Users\\rdboylan\\Documents\\KBD2\\CVDPM Inputs\\shortfile_99_14.sas7bdat"))
nhs = broadcast(string, names(nh))
wts = nhs[broadcast(x-> occursin(r"wt|wgt|weight"i, x), nhs)]

# step 5c
moreCat = [ :dbpcat, :diabetcat2, :CKDCAT, :Statin,  :htnmedscat, :CVDCAT3, :CVDcat, :PACAT2,
    :SESCAT ]
```

I do not even know for sure the error came from Julia.

```julia
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = "C:\Users\rdboylan\AppData\Local\atom\app-1.41.0\atom.exe"  -a
  JULIA_NUM_THREADS = 6
```

**Comment by RossBoylan** [comment]
FWIW I repeated the same steps in a vanilla Julia session (console not atom) and there was no crash.

I repeated them in atom, and there was no crash.  I attempted to repeat them exactly, and so I used ctl-Enter to evaluate the first 2 blocks (5a and b) and alt-enter to evaluate the last (5c).  This strongly suggested alt-enter reevaluated the entire buffer, and so my statement that the crash followed assignment of a list of symbols to a variable may not be correct.  It certainly isn't what happened the second time.  I think shift-enter was what I was trying for (evaluate and move to next).
